### ISSUE TITLE
RectangleBarSeries: wrong tracker values after zooming in

### ISSUE THREAD

**Issue by bac82** [oxyplot/oxyplot#1171]
### Steps to reproduce

1. Create a plotmodel with RectangleBarSerie and RectangleBarItems
2. Using default `TrackerFormatString = "{0}\n{1}: {2} {3}\n{4}: {5} {6}";`
3. Start Application
4. Zooming in with "+"-Button on Keypad for example
5. Mouse-over or click (depending on Plotcontroller) on RectangleBarItem to show tracker window

Platform: Windows 7 / WPF
.NET version: 4.6.1
OxyPlot.Core/.WPF v1.0.0

### Expected behaviour

Tracker is showing the correct values of RectangleBarItem.

### Actual behaviour

Tracker is showing incorrect values:

![rectanglebarseries_tracker_error](https://user-images.githubusercontent.com/17470778/33321608-9003a432-d446-11e7-97e7-6a5786f066a8.png)

Visual Studio 2015 sample project:

[OxyApp1_TrackerIssue.zip](https://github.com/oxyplot/oxyplot/files/1509993/OxyApp1_TrackerIssue.zip)

The plotmodel I used for this report:

        public static PlotModel getPlotModel()
        {
            var plotModel1 = new PlotModel();
            plotModel1.PlotMargins = new OxyThickness(50, Double.NaN, 10, Double.NaN);
            var linearAxis1 = new LinearAxis();
            linearAxis1.Position = AxisPosition.Bottom;
            linearAxis1.Minimum = 0;
            linearAxis1.Maximum = 200;
            plotModel1.Axes.Add(linearAxis1);
            var linearAxis2 = new LinearAxis();
            linearAxis2.AbsoluteMinimum = 0;
            //linearAxis2.IsPanEnabled = false;
            linearAxis2.MajorGridlineStyle = LineStyle.Solid;
            linearAxis2.Maximum = 60;
            linearAxis2.Minimum = 0;
            linearAxis2.MinorGridlineStyle = LineStyle.Dot;
            linearAxis2.Title = "speed (m/s)";
            plotModel1.Axes.Add(linearAxis2);
            var rectangleBarSeries1 = new RectangleBarSeries();
            rectangleBarSeries1.FillColor = OxyColors.Red;
            rectangleBarSeries1.StrokeThickness = 0.5;
            rectangleBarSeries1.Title = "max";

            rectangleBarSeries1.TrackerFormatString = "{0}\n{1}: {2} {3}\n{4}: {5} {6}"; //default

            rectangleBarSeries1.Items.Add(new RectangleBarItem(20.5, 0, 24.5, 20.000));
            rectangleBarSeries1.Items.Add(new RectangleBarItem(25.5, 0, 29.5, 22.000));
            rectangleBarSeries1.Items.Add(new RectangleBarItem(30.5, 0, 34.5, 23.000));
            rectangleBarSeries1.Items.Add(new RectangleBarItem(35.5, 0, 39.5, 24.000));
            rectangleBarSeries1.Items.Add(new RectangleBarItem(40.5, 0, 44.5, 25.000));
            rectangleBarSeries1.Items.Add(new RectangleBarItem(45.5, 0, 49.5, 26.000));
            rectangleBarSeries1.Items.Add(new RectangleBarItem(50.5, 0, 54.5, 27.000));
            rectangleBarSeries1.Items.Add(new RectangleBarItem(55.5, 0, 59.5, 28.000));
            rectangleBarSeries1.Items.Add(new RectangleBarItem(60.5, 0, 64.5, 29.000));
            rectangleBarSeries1.Items.Add(new RectangleBarItem(65.5, 0, 69.5, 30.000));
            rectangleBarSeries1.Items.Add(new RectangleBarItem(70.5, 0, 74.5, 31.000));
            rectangleBarSeries1.Items.Add(new RectangleBarItem(75.5, 0, 79.5, 32.000));
            rectangleBarSeries1.Items.Add(new RectangleBarItem(80.5, 0, 84.5, 33.000));
            rectangleBarSeries1.Items.Add(new RectangleBarItem(85.5, 0, 89.5, 34.000));
            rectangleBarSeries1.Items.Add(new RectangleBarItem(90.5, 0, 94.5, 35.000));
            rectangleBarSeries1.Items.Add(new RectangleBarItem(95.5, 0, 99.5, 36.000));

            plotModel1.Series.Add(rectangleBarSeries1);
            return plotModel1;
        }

**Comment by bac82** [comment]
Same issue with OxyPlot.Core/.WPF packages **v2.0.0-unstable1013**

**Comment by bac82** [comment]
I took a closer look. `RectangleBarSeries.cs` has an issue in the method  `GetNearestPoint()`:

The `TrackerHitResult` is obtained from `this.Items[i]`, where `Items` is the wrong list (contains all items) when zooming into the plot. Similar to `ActualBarRectangles` I created a `ActualItems = List<RectangleBarItem>` in the method `Render()` and use this list in `GetNearestPoint()` to create the `TrackerHitResult`.

The tracker now shows correct values for the selected `RectangleBarItem`.
